EosGround/config/eos_schema.sql

```diff
@@ -16,23 +16,23 @@ BEGIN;
 
 CREATE TABLE IF NOT EXISTS eos_schema.received_data
 (
-    id bigint NOT NULL GENERATED ALWAYS AS IDENTITY,
```

Comment:
yeah agreed,  i dont think we'll ever exceed 2 billion packets, we can up the size later if needed